tools/docker-boring/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM golang:1.19.1-alpine as builder
+
+RUN apk add --no-cache git build-base
+
+RUN cd /go/src && git clone https://github.com/kastenhq/kopia
+
+WORKDIR /go/src/kopia
+
+ARG kopiaCommit="boring_kopia"
+
+RUN git checkout "${kopiaCommit}"
+
+
+RUN GO111MODULE=on GO_EXTLINK_ENABLED=0 GOOS=linux GOARCH=amd64 CGO_ENABLED=1 GOEXPERIMENT=boringcrypto go build -o kopiaboring \
```

Comment:
BoringCrypto is CGO, this is going to perform worse than the default crypto lib. CGO imposes a fix 200ns impact for the context switch to CGO for any particular function call. Is that acceptable ?

Simple calls to encrypt() would go from 10ns to 200ns immediately with this change. We might need to reduce the number of calls we make to boringcrypto from golang to make this less impactful.